Libplanet.Net/Messages/ConsensusVote.cs

```diff
@@ -1,19 +1,35 @@
-using Libplanet.Blocks;
+using System.Collections.Generic;
+using Libplanet.Consensus;
 
 namespace Libplanet.Net.Messages
 {
     public class ConsensusVote : ConsensusMessage
     {
-        public ConsensusVote(long nodeId, long height, long round, BlockHash blockHash)
-            : base(nodeId, height, round, blockHash)
+        public ConsensusVote(Vote vote)
+            : base(vote.NodeId, vote.Height, vote.Round, vote.BlockHash)
         {
+            ProposeVote = vote;
         }
 
         public ConsensusVote(byte[][] dataframes)
-            : base(dataframes)
+            : this(new Vote(dataframes[0]))
         {
         }
 
+        public Vote ProposeVote { get; }
+
+        public override IEnumerable<byte[]> DataFrames
+        {
+            get
+            {
+                var frames = new List<byte[]>
+                {
+                    ProposeVote.ByteArray,
+                };
+                return frames;
```

Comment:
May be missing `base.DataFrames`?

Reply:
https://github.com/planetarium/libplanet/pull/1979/files#diff-8f7496d7becdca3f0362db3cbd3c09f4c0cc16923436b0f4aeec5e094be984bcR14-R15 You should check out here. base.DataFrames is unnecessary. it initialized on the constructor.